openbook_auth/models.py

```diff
@@ -1445,23 +1445,23 @@ def get_community_post_with_id(self, post_id):
 
         post_query.add(~Q(community__banned_users__id=self.pk), Q.AND)
 
+        # Check we have membership or its a public community
         post_query_visibility_query = Q(community__memberships__user__id=self.pk)
         post_query_visibility_query.add(Q(community__type=Community.COMMUNITY_TYPE_PUBLIC, ), Q.OR)
-        post_query_visibility_query = Q(community__memberships__user__id=self.pk, is_closed=False)
```

Comment:
will this not filter all posts where user is member or the community is public ( these should be ANDed with is_closed=False) .. is return _only_ open posts where user is member or community is public

Reply:
You commented in your own changes 🤔

Reply:
This line is gone with my changes

Reply:
yea weird, i commented on ur commit.. but i see it at both places